### ISSUE TITLE
[BUG] Custom TW class not recognized

### ISSUE THREAD

**Issue by asharonbaltazar** [francoismassart/eslint-plugin-tailwindcss#51]
**Describe the bug**
Custom TW classes are being underlined. 

**To Reproduce**
1. On `1.14.3`
2. `extend` Tailwind colors
3. Error still shown

**Expected behavior**
It should recognize the classes from the `tailwind.config.js`

**Screenshots**
![image](https://user-images.githubusercontent.com/58940073/133831072-7a91f9c8-2f95-4ca3-892f-64b2503a5f2e.png)

**Environment (please complete the following information):**
 - OS: Windows 10
 - Softwares + version used:
     -  VSCode (latest)
     -  npm 6.14.13, node v14.17

**Additional context**
I've also added a `config` property to the `eslint` rule. That didn't help

**Comment by francoismassart** [comment]
Hi @asharonbaltazar, it looks like the plugin does not get the `config.tailwind.js` file...

I can help if you are willing to set up a dummy project demonstrating the issue, as simple as possible as long as it demonstrate the problem you are experiencing.

**Comment by asharonbaltazar** [comment]
Okay, while I try to recreate the env, here's some code. Perhaps the error is obvious:

**tailwind.config.js**
```javascript
const defaultTheme = require('tailwindcss/defaultTheme');
const colors = require('tailwindcss/colors');

module.exports = {
    ...
    theme: {
        extend: {
            ...
            textColor: {
                primary: '#48466D',
                ...
                alt: '#707070',
            },
        },
        plugins: [],
        colors: {
            ...defaultTheme.colors,
            transparent: 'transparent',
            ...
            lime: colors.lime,
        },
    },
};
```

**.eslintrc.js**
```javascript
module.exports = {
    root: true,

    settings: {
        ...
        react: {
            version: 'detect',
        },
        tailwindcss: {
            callees: ['classNames'],
        },
    },
    plugins: ['@typescript-eslint', 'prettier', 'tailwindcss'],
    extends: ['airbnb', 'prettier', 'plugin:tailwindcss/recommended'],
    parser: '@typescript-eslint/parser',
    ...
    rules: {
              'tailwindcss/no-custom-classname': [
                  'error',
                  {
                      config: tailwind.config.js,
                      whitelist: ['abcRioButtonSvg'],
                  },
              ],
              ...
          },
};
```

**Comment by asharonbaltazar** [comment]
Maybe it's a workaround, but I did `require` the file and used that (`const tailwind = require('./tailwind.config')`) instead of a `path` for the `config` property. It worked!

**Comment by francoismassart** [comment]
Glad you find a solution for your issue.

As indicated in the docs, the [config](https://github.com/francoismassart/eslint-plugin-tailwindcss/blob/master/docs/rules/classnames-order.md#config-default-tailwindconfigjs) can take a string (which is considered to be a path to the config to be loaded, or a plain object):
> It is also possible to directly inject a configuration as plain object like `{ prefix: "tw-", theme: { ... } }`.

In your case, requiring the file passes a plain object containing the tailwind config 🎉
